Assets/BossRoom/Scripts/Shared/Net/GameNetPortal.cs

```diff
@@ -56,10 +56,10 @@ public class GameNetPortal : MonoBehaviour
         public GameObject NetworkManagerGO;
 
         /// <summary>
-        /// This synthesizes a general NetworkStart event out of other events provided by MLAPI. This can be removed
-        /// when the NetworkManager starts publishing this event directly.
+        /// This event is fired when MLAPI has reported that it has finished initialization, and is ready for
+        /// business, equivalent to OnServerStarted on the server, and OnClientConnected on the client.
         /// </summary>
-        public event Action NetworkStarted;
+        public event Action NetworkReadyEvent;

```

Comment:
"- Include one participle-form verb in the event name (generally ending in -ed or -ing, ex. occurred, loading, started, given)" 

"NetworkReadied" maybe?

Reply:
sounds good to me. updated.